src/calculateRentalCost.js

```diff
@@ -28,7 +28,22 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let price;
+  const rent = 40;
+
+  if (days >= 1) {
+    price = days * rent;
```

Comment:
set this as the initial price value - that will allow you to get rid of this if statement (and just subtract discount in other if blocks)